### DISCUSSION TITLE
Missing user in docker container

### DISCUSSION THREAD

**Discussion by solairen** [JanDeDobbeleer/oh-my-posh#3691] [Troubleshoot] [answered]
Hi,

I have a problem with a missing user in a docker container.

I installed oh-my-posh on my Windows and Linux and the theme works great. I can see the user and other stuff that I added.

But, when I installed oh-my-posh inside a docker container and paste there my theme the user is missing.

I tried with the default root user inside a container and also with a created user.

On my Windows

```
fancyUser on Friday at 11:59 AM
```

Inside Docker Container
```
  on Friday at 10:03 AM
```

Do I need to add some extra arguments to a docker container?

**Comment by JanDeDobbeleer** [comment, marked as answer]
We fetch the `USER` and `USERNAME` environment variable. If both are empty, there's no value. But that's rather strange TBH.

**Reply by solairen** [reply]
Thanks a lot! After adding those environment variables it starts showing the user.
